addons/l10n_sa_edi/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+l10n_sa_edi_otp_wizard,l10n_sa_edi_otp_wizard,model_l10n_sa_edi_otp_wizard,base.group_user,1,1,1,1
+l10n_sa_edi_submission,l10n_sa_edi_submission,model_l10n_sa_edi_submission,base.group_user,1,1,1,1
```

Comment:
should be account.group_account_invoice instead and I am even not sure a user needs to change anything here, so best 1,0,0,0

Reply:
Done